platform/security/expansion/security-expansion-metacardattr-map/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -18,14 +18,13 @@
     <OCD description="Expansion Service for Metacard Attributes"
          name="Security Metacard Attribute Mapping"
          id="ddf.security.metacard.attribute.mapping">
-        <AD description="Attribute separator - used to separate attributes in replacement strings - defaults to space (specified by no value)."
-            name="Attribute Separator" id="attributeSeparator" required="false"
-            type="String" default=" "/>
+        <AD description="Used to separate attributes in replacement strings. Defaults to space (specified by no value)."
```

Comment:
Is there a doc table that should be updated as well?

Reply:
I couldn't find one.